.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,32 +9,35 @@ assignees: ''
 
 <!-- Please search for open issues that relate to the same problem before opening a new one. -->
 
-### Describe the bug:
+### Bug Description
```

Comment:
Can we just fall this ”Description” instead?

Reply:
Agreed, will change this.